python/pyspark/sql/streaming/StateMessage_pb2.pyi

```diff
@@ -0,0 +1,139 @@
+from google.protobuf.internal import enum_type_wrapper as _enum_type_wrapper
```

Comment:
@bogao007 @sahnib - i have a general question. how do we choose between Protobufv2 and Protobufv3 ?

Reply:
Spark Connect was using proto3 when introducing new proto messages, I was trying to follow the same. I was not able to find a guide in OSS Spark on which version is recommended though. cc @HyukjinKwon

Reply:
Oh ok - the generated name was `pb2` - so I thought its using proto2